avm/res/compute/disk-encryption-set/tests/e2e/cmk-hsm-uami/main.test.bicep

```
@@ -0,0 +1,85 @@
+targetScope = 'subscription'
+
+metadata name = 'Using managed HSM Customer-Managed-Keys with User-Assigned identity'
+metadata description = 'This instance deploys the module with Managed HSM-based Customer Managed Key (CMK) encryption, using a User-Assigned Managed Identity to access the HSM key.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-compute.diskencryptionsets-${serviceShort}-rg'
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'cdeshsm'
```

```suggestion
param serviceShort string = 'cdeshsmu'
```
Small thing, but I started using `u` as a suffix as it would be different for system-assigned (which could then have `s` as a suffix). Won't block the PR over it, but it made sense to me